index.html

```diff
@@ -100,23 +110,40 @@ <h2>
     </div>
   </div>
 
-  Phone: +1-543-123-4567
-  example@fylo.com
-
-  About Us
-  Jobs
-  Press
-  Blog
-
-  Contact Us
-  Terms
-  Privacy
-
   <footer>
-    <p class="attribution">
-      Challenge by <a href="https://www.frontendmentor.io?ref=challenge" target="_blank">Frontend Mentor</a>. 
-      Coded by <a href="#">Your Name Here</a>.
-    </p>
-  </footer>
-</body>
-</html>
\ No newline at end of file
+    <nav>
+      <div class="contact-info">
+        <img src="images/footer.svg" alt="Fylo" class="logo-image">
+        <p><img src="images/icon-phone.svg" alt="Phone Icon" class="icon"><font color="#ffffff">Phone: +1-543-123-4567</font> </p>
+        <p><img src="images/icon-email.svg" alt="Email Icon" class="icon"><font color="#ffffff"> Email: example@fylo.com </font></p>
```

Comment:
It is not good practice to nest an image within a paragraph, instead use a container element such as div..